scanner/updater/export.go

```diff
@@ -217,3 +219,26 @@ func (t *rateLimitedTransport) RoundTrip(req *http.Request) (*http.Response, err
 	}
 	return t.transport.RoundTrip(req)
 }
+
+func epssOpts(ctx context.Context) []updates.ManagerOption {
+	enricher := &epss.Enricher{}
+	configJSON, err := json.Marshal(epss.Config{
+		URL: nil, // use default URL for now
+	})
+	if err != nil {
+		log.Printf("Failed to encode EPSS enricher config : %v", err)
+		return nil
+	}
+
+	err = enricher.Configure(ctx, func(config interface{}) error {
+		return json.Unmarshal(configJSON, config)
+	}, &http.Client{})
+	if err != nil {
+		log.Printf("Failed to configure EPSS enricher: %v", err)
+		return nil
+	}
+	return []updates.ManagerOption{
+		updates.WithEnabled([]string{}),
```

Comment:
Please add `// This is required to prevent default updaters from running.` so future readers know why this is there.